Description: Disable GIT based version calculation
Author: Markus Frosch <markus@lazyfrosch.de>
Bug: https://dev.icinga.org/issues/5410
Last-Update: 2013-12-29
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -47,14 +47,14 @@
 set(CPACK_SOURCE_IGNORE_FILES "/.git/")
 include(CPack)
 
-include(GetGitRevisionDescription)
-git_describe(GIT_VERSION --tags)
-if(GIT_VERSION MATCHES "-NOTFOUND$")
+#include(GetGitRevisionDescription)
+#git_describe(GIT_VERSION --tags)
+#if(GIT_VERSION MATCHES "-NOTFOUND$")
   configure_file(icinga-version.h.fallback ${CMAKE_CURRENT_BINARY_DIR}/icinga-version.h COPYONLY)
-else()
-  configure_file(icinga-version.h.cmake icinga-version.h)
-  configure_file(${CMAKE_CURRENT_BINARY_DIR}/icinga-version.h ${CMAKE_CURRENT_SOURCE_DIR}/icinga-version.h.fallback COPYONLY)
-endif()
+#else()
+#  configure_file(icinga-version.h.cmake icinga-version.h)
+#  configure_file(${CMAKE_CURRENT_BINARY_DIR}/icinga-version.h ${CMAKE_CURRENT_SOURCE_DIR}/icinga-version.h.fallback COPYONLY)
+#endif()
 
 if(WIN32)
   set(Boost_USE_STATIC_LIBS ON)
